vuu-ui/packages/vuu-shell/src/layout-management/LayoutList.css

```diff
@@ -1,38 +1,70 @@
 .vuuLayoutList {
+    --vuuLayoutList-width: 90%;
     --vuuMeasuredContainer-flex: 1 1 1px;
     align-self: stretch;
     display: flex;
     flex-direction: column;
     height: 100%;
     overflow: hidden;
+    padding-right: 6px;
+
+    /* Works on Firefox */
+    scrollbar-width: thin;
+    scrollbar-color: #D9D9D9 transparent;
+
+    /* compatible with Chrome, Edge, and Safari */
+    ::-webkit-scrollbar {
+        width: 4px;
+        height: 4px;
+    }
```

Comment:
What does the height here apply to? Horizontal scrollbars?

Reply:
Yep.